pkg/app/app.go

```diff
@@ -107,21 +107,20 @@ func GetFromSlug(slug string) (*App, error) {
 
 // UpdateConfigValuesInDB it gets the config values from filesInDir and
 // updates the current sequence config values in the db
-// THIS SHOULD ONLY BE CALLED ON APP INSTALLS (WHEN CREATING THE FIRST VERSION)
-func (a App) UpdateConfigValuesInDB(filesInDir string) error {
+func (a App) UpdateConfigValuesInDB(filesInDir string, sequence int64) error {
```

Comment:
Since this has changed, let's update the comment to be accurate and I don't think it should be a member of App struct now

Reply:
About removing it from the app struct, we still need the struct to get the app id, no @marccampbell ?

Reply:
I think this would be better just passed as param then. It's not using the app struct, and it's not a pointer/modifying the struct. It would be cleaner IMO as a standalone function.

Reply:
Got it, will update